src/Sylius/Bundle/AdminBundle/Resources/config/validation/PasswordResetRequest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+This file is part of the Sylius package.
+
+(c) Paweł Jędrzejewski
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+-->
+
+<constraint-mapping xmlns="http://symfony.com/schema/dic/constraint-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/constraint-mapping http://symfony.com/schema/dic/services/constraint-mapping-1.0.xsd">
+    <class name="Sylius\Bundle\AdminBundle\Form\Model\PasswordResetRequest">
+        <property name="email">
+            <constraint name="NotBlank" />
+            <constraint name="Email" />
+        </property>
+    </class>
```

Comment:
Is there an implemented scenario for this validation?

Reply:
Nope, I've split the main task a little bit too much and validation will be covered later.